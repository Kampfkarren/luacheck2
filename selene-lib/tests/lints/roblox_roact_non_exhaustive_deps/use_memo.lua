local React

local function Component(props)
    React.useEffect(function()
        print(props)
    end, {})
end
