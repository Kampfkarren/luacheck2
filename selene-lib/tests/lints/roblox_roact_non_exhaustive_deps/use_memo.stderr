error[roblox_roact_non_exhaustive_deps]: react hook useEffect has missing dependency: 'props'
  ┌─ use_memo.lua:6:10
  │
6 │     end, {})
  │          ^^
  │
  = help: either include it or remove the dependency array

